iml-api/policy.csv

```diff
@@ -0,0 +1,86 @@
+p, superusers               , query::client_mount_command
```

Comment:
Michael, why has CSV format been chosen? YAML allows e.g. to insert comments and to structure the description as a map:
```
"query::client_mount_command":
  - superusers
  - filesystem_administrators
  - filesystem_users
```

Reply:
It is what casbin uses out of the box. Not sure if they support anything else.

Reply:
Casbin YAML adapter is done, see: https://github.com/casbin/casbin-rs/issues/213#issuecomment-765388219